### PR TITLE
[WIP] Rubocop:Metrics/LineLength

### PR DESCRIPTION
**Pull request by jaredmoody**
This is a first-pass at the line-lengths cop - I set it to 100 and corrected all warnings and then wanted to get any feedback from you guys before the next pass. 

I figure I'll then set it to 80, run and correct any errors that don't sacrifice readability and then set back to 100 for the final commit.

Let me know if things look good and how that general approach sounds.

### CODE REVIEW

File: benchmark/para_creator.rb [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
+    "request.) So you’re like, “Wow, this book comes with an onion!” " \
+    "(Even if you don’t particularly like onions, I’m sure you can " \
+    "appreciate the logistics of shipping any sort of produce discreetly " \
+    "inside of an alleged programming manual.) Then you ask yourself, " \
+    "“Wait a minute. I thought this was a book on Ruby, the incredible new " \
+    "programming language from Japan. And although I can appreciate the " \
+    "logistics of shipping any sort of produce discreetly inside of an " \
+    "alleged programming manual: Why an onion? What am I supposed to do with " \
+    "it?” No. Please don’t puzzle over it. You don’t need to do anything " \
+    "with the onion. Set the onion aside and let it do something with you."
```
Comment (jasonrclark) [inline benchmark/para_creator.rb]: I've got a slight inclination toward using [heredoc style strings](https://ruby-doc.org/core-2.2.0/doc/syntax/literals_rdoc.html#label-Here+Documents), but totally agree that the multi-line here is a huge boost to readability.

File: samples/expert_tetris.rb [hunk 2]
```diff
@@ -63,9 +100,13 @@ class Tetris
...
               :score,    # the current score
               :lost,     # bool to indicate when the game is lost
-              :pace,     # current game pace - how long until the current piece drops a single row
-              :blocks,   # 2 dimensional array (NX*NY) represeting the tetris court - either empty block or occupied by a piece
-              :actions,  # queue of user inputs collected by the game loop
+              :pace      # current game pace - how long until the current piece drops a single row
+
+  # 2 dimensional array (NX*NY) represeting the tetris court - either empty block or occupied
+  # by a piece
+  attr_reader :blocks
```
Comment (jasonrclark) [inline samples/expert_tetris.rb]: Would rather we didn't unnecessarily repeat the `attr_reader` just to get the commenting above the items... bet we could just dangle the commas before and allow comments between items generally so the whole section's consistent.

**Comment by jasonrclark** [comment]
These are looking reasonable to me. As expected, most of the places where things are that long deserve some attention and tightening anyway.

Looks like tests are unhappy, but I haven't glanced at why. You might want to check those before proceeding further, but I'm :+1: on continuing down this track. Thanks @jaredmoody!

**Comment by PragTob** [comment]
:wave:

What's the state here? :)

**Comment by jaredmoody** [comment]
The state is life events :P It's going to be a couple weeks before I can finish this off.

**Comment by jasonrclark** [comment]
Looks like this has developed a fairly large collection of merge conflicts while it's been sitting. Shall we just close and reapply the stuff whenever you or someone else wants to get to it @jaredmoody ?

**Comment by jaredmoody** [comment]
Sounds good to me 👍🏼

**Pull request closed without merging**